nasa/ow_autonomy
# PLEXIL uses pan/tilt ROS action

Summary:

- PLEXIL now uses the ROS action for pan/tilt instead of commanding these using topics.
- All PLEXIL plans that had commanded Pan and Tilt now use the combined Pan/Tilt plan.
- The implementation of the non-action Pan and Tilt is completely removed.

Unrelated changes:
 - Fixed some incomplete maps in OwInterface.cpp.
 - Removed some dead C++ code.
 - Changed one or a few INFO messages to DEBUG to quiet the output.
 - Many file lines have trailing whitespace removed (a default Emacs feature).

Test:
 - NOTE: this branch is best rebuilt clean:
      `$ rm -r <ow-workspace>/build/ow_plexil`
      `$ catkin build ow_plexil`
 - A clean build itself verifies many of the changes.
 - Start any simulator world.
 - Run the `TestAntennaMovement` plan, e.g.
      `$ roslaunch ow_plexil ow_exec.launch plan:=TestAntennaMovement.plx`
      It should completely pass (13/13).
 - Run ReferenceMission1, for regression (because of unrelated code changes)
 - Optionally, run any other plans that use the antenna (I've tested all of them).

Notes:
 - Don't be scared by the number of changed files -- most of the changes are trivial.
 - There is a known bug, unrelated to this work, that will manifest in any plan that takes images.  The following sequence will appear at intervals.
   ` WARN]: Fault in SYSTEM: CAMERA_EXECUTION_ERROR`
    `[WARN]: Resolved fault in SYSTEM: CAMERA_EXECUTION_ERROR`
 - @AstroStucky This builds on the PR you just reviewed.
 - @mogumbo The main test shows examples of mismatching pan goals/results.
